contracts/child/ChildERC721Mintable.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity ^0.5.2;
+
+import { ChildERC721 } from "./ChildERC721.sol";
+import { ERC721Mintable } from "openzeppelin-solidity/contracts/token/ERC721/ERC721Mintable.sol";
+
+contract ChildERC721Mintable is ChildERC721, ERC721Mintable {
+  constructor (address rootToken)
+    ChildERC721(msg.sender, rootToken, "Mintable 721", "M721")
```

Comment:
Who will be the minter?

Reply:
The authorized minters fn the root chain 721 contract! The minters on root will need to be added to `ChildERC721Mintable.addMinter(address account)` (mapped contract on child)

Reply:
Got it.